worker/runtime/spec/spec.go

```diff
@@ -150,7 +151,9 @@ func OciResources(limits garden.Limits) *specs.LinuxResources {
 	if memoryLimit > 0 {
 		memoryResources = &specs.LinuxMemory{
 			Limit: &memoryLimit,
-			Swap:  &memoryLimit,
+		}
+		if IsSwapLimitEnabled {
+			memoryResources.Swap = &memoryLimit
```

Comment:
I'm not a fan of making global variables public purely for testing, can we have `OciSpec` pass this in as a parameter instead?